docs/conf.py

```diff
@@ -138,19 +142,20 @@
 # Add any paths that contain custom themes here, relative to this directory.
 # To use a different custom theme, add the directory containing the theme.
 #html_theme_path = []
-
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes. To override the custom theme, set this to the
 # name of a builtin theme or the name of a custom theme in html_theme_path.
-#html_theme = None
+html_theme = 'bootstrap-astropy'
+
+html_theme_options = {'astropy_project_menubar': True}
```

Comment:
html_theme_options might already be defined in the sphinx-astropy config so you may need to do ``html_theme_optons['...'] = ...`` here, but not 100% sure.

Reply:
Yes, `html_theme_options` is already defined in the sphinx-astropy config... am figuring out what to try

Reply:
But in hindsight I think if I set `html_theme_options = {'astropy_project_menubar': True}` in the conf.py in the astropy/docs directory it will override the setting in the `sphinx-astropy` config. Just did a `git clean -dfx` and rebuild the docs locally again, should take no more than 30 minutes...